client-app/core/utilities/index.ts

```diff
@@ -9,3 +9,4 @@ export * from "./menu";
 export * from "./product";
 export * from "./properties";
 export * from "./search";
+export * from "./tests";
```

Comment:
try remove it then error form ci should disappeared

Reply:
didn't help, found a workaround with passing dependency into utility function from the test file.